Gemfile

```diff
@@ -1,11 +1,12 @@
+# frozen_string_literal: true
```

Comment:
In Wolox, we don't use this line, instead you could add the following line into the  rubocop.yml to avoid that warning:

```
Style/FrozenStringLiteralComment:
  Enabled: false

```

That way, you don't have to include  that line in ALL the files

Reply:
Done